src/ets/scala/io/iohk/ethereum/ets/blockchain/BlockchainSuite.scala

```diff
@@ -0,0 +1,97 @@
+package io.iohk.ethereum.ets.blockchain
+
+import io.iohk.ethereum.ets.common.TestOptions
+import io.iohk.ethereum.ledger.BlockExecutionError
+import io.iohk.ethereum.network.p2p.messages.CommonMessages.NewBlock
+import io.iohk.ethereum.utils.Logger
+import org.scalatest._
+
+
+class BlockchainSuite extends FreeSpec with Matchers with Logger {
+
+  val unsupportedNetworks = Set("Byzantium","Constantinople", "EIP158", "EIP158ToByzantiumAt5", "HomesteadToDaoAt5")
+  val supportedNetworks = Set("EIP150", "Frontier", "FrontierToHomesteadAt5", "Homestead", "HomesteadToEIP150At5")
+
+  type GroupName = String
+  type TestName = String
+  type IgnoredTestNames = Set[String]
```

Comment:
What's the purpose of these types?

Reply:
actually, they help with understanding what is this map :
` val ignoredTests: Map[GroupName, IgnoredTestNames]`
but i don't stick to them if you think it is unnecessary